src/generic_core/user.spec.ts

```diff
@@ -95,20 +95,6 @@ describe('Core.User', () => {
     expect(network.sendInstanceHandshake).not.toHaveBeenCalled();
   });
 
-  it('does not send instance messages to non-uProxy clients', () => {
```

Comment:
I think this test, rather than being removed, should now be that `ONLINE_WITH_OTHER_APP` clients are not added to the `user.clientIdToStatusMap`.